tools/db/replicateDbs.py

```diff
@@ -25,12 +25,24 @@
 import re
 import couchdb.client
 
+def retry(fn, retries, *args):
+    try:
+        return fn(*args)
+    except:
+        if (retries > 0):
+            time.sleep(1)
+            return retry(fn, retries - 1, *args)
+        else:
+            raise
```

Comment:
As noted in chat, `retry` should take an applied function (`Function0`) achieveable for example using `lambda`.